src/Todo.tsx

```
@@ -0,0 +1,96 @@
+      />
+      <Button onClick={addTodo} type="primary">
+        Add
+      </Button>
+      <List
+        style={{ marginTop: "20px" }}
+        bordered
+        dataSource={todos}
+        renderItem={(item) => (
+          <List.Item
+            actions={[
+              <a onClick={() => toggleComplete(item.id)}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)